fleetspeak/src/client/internal/config/manager.go

```diff
@@ -74,27 +68,23 @@ func StartManager(cfg *config.Configuration, configChanges chan<- *fspb.ClientIn
 		return nil, errors.New("configuration must be provided")
 	}
 
+	if cfg.PersistenceHandler == nil {
```

Comment:
I'd probably plug in a NoopPersistenceManager in this case, and simply leave it unset in tests. This matches the behavior of the server's stats collector: https://github.com/google/fleetspeak/blob/8c84e6c3234ae78199e0cbf071dbfa46eb38bb65/fleetspeak/src/server/server.go#L63

I do see that it increases the odds of somebody not setting it when needed, but Config object already has plenty of things that must be set and I think it is fine so long as the docs make it clear what to set and miniclient, etc, are correct.. We could also log a warning here when it is nil. 

This may be a question of taste though.

Reply:
A question of taste indeed - I also like the proposed version better. Done